src/Styleguide/Components/__stories__/StepSummaryItem.story.tsx

```diff
@@ -0,0 +1,50 @@
+import { Sans, Serif } from "@artsy/palette"
+import React from "react"
+import { storiesOf } from "storybook/storiesOf"
+import { StepSummaryItem } from "Styleguide/Components/StepSummaryItem"
+import { Section } from "Styleguide/Utils/Section"
+import { Responsive } from "Utils/Responsive"
+
+storiesOf("Styleguide/Components", module).add("StepSummaryItem", () => {
+  return (
+    <>
+      <Section title="Single StepSummaryItem">
+        <StepSummaryItem
+          title="Pick up (New York, NY)"
+          onChange={() => console.warn("change")}
+        >
+          <Sans size="2" color="black60" maxWidth={391}>
+            After you place your order, you’ll be appointed an Artsy specialist
+            within 2 business days to handle pickup logistics.
+          </Sans>
+        </StepSummaryItem>
+      </Section>
+      <Section title="Multiple StepSummaryItems">
+        <StepSummaryItem
+          title="Pick up (New York, NY)"
+          onChange={() => console.warn("change")}
+        >
+          <Sans size="2" color="black60" maxWidth={391}>
```

Comment:
This one too?

Reply:
Also Zeplin

Reply:
I left this hard-coded in the story file for now. It's not clear to me that this constraint should come from the grid system, because it's not about alignment so much as it is about upholding the legibility of copy that spans multiple lines.

My guess is that it should probably be defined in palette based on the font size? But overridable for situations where having longer lines is the desired behaviour, of course.